### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Achilles-Dev**
I created a todo list that has the following functionality:
- [ ] An input field to create and add a todo-list item and a function to store the item in localStorage
- [ ] A function to edit the created todo-list and update the localStorage with the edited item.
- [ ] A function to clear all completed todo-list items when the `clear all completed button` is clicked.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  <main>
+    <div class="title">
+      <h3>Today's To Do</h3>
+      <img src="#" alt="Refresh Icon" class="refresh-icon">
```
Comment (Theophileaseh) [inline src/index.html:13]: Image source not found

File: README.md [hunk 2]
```diff
@@ -1,56 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Theophileaseh) [inline README.md:1]: please update your README file

File: README.md [hunk 3]
```diff
@@ -1,56 +1,36 @@
...
+👤 **Solomon Hagan**
 
-👤 **Author2**
+- GitHub: [@Achilles-Dev](https://github.com/Achilles-Dev/)
+- Twitter: [@SoloAchilles](https://twitter.com/SoloAchilles/)
+- LinkedIn: [Solomon Hagan](https://www.linkedin.com/in/solomon-hagan-b51693138/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (Theophileaseh) [inline README.md:53]: please update your README file

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  background: #f6f6f6;
+}
+
+main {
+  display: flex;
```
Comment (Theophileaseh) [inline src/style.css:9]: You can make your code DRY by bringing all the display: flex, together

**Review by Theophileaseh — COMMENTED** [review]
Hello, your code is clean and done in a professional way. Could you please go through the following suggestions

File: src/index.html (hunk 1 above)
Comment (Achilles-Dev) [inline src/index.html:13]: The `src` attribute name `#` is replaced dynamically with javascript.

**Comment by Theophileaseh** [comment]
Ok

File: src/index.html (hunk 1 above)
Comment (Theophileaseh) [inline src/index.html:13]: Image link missing

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <form action="#" method="POST" class="input-form">
+      <input type="text" name="description" placeholder="Add to your list...">
+      <img src="#" alt="Enter Icon" class="enter-icon">
```
Comment (Theophileaseh) [inline src/index.html:17]: Image link missing

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  <main>
+    <div class="title">
+      <h3>Today's To Do</h3>
+      <img src="#" alt="Refresh Icon" class="refresh-icon">
+    </div>
+    <form action="#" method="POST" class="input-form">
```
Comment (Theophileaseh) [inline src/index.html:15]: No need to add `action`

**Review by Theophileaseh — COMMENTED** [review]
Hi @Achilles-Dev 
Almost There


While you have made a great effort to complete this project, implement the above suggested changes for everything to be ok.

**Thanks
Happy Coding! 💻**

**Pull request merged**